scram-tools.file/tools/libbsd/libbsd.xml

```diff
@@ -0,0 +1,8 @@
+<tool name="LIBBSD" version="@TOOL_VERSION@">
```

Comment:
@iarspider , can you please use lower-case too name i.e. `libbsd` and `libmd`

Reply:
Done